producer-model/src/main/scala/za/co/absa/spline/producer/model/v1_1/AttrOrExprRef.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.producer.model.v1_1
+
+/**
+ * Represents expression or attribute ID in untyped or weakly typed data structures,
+ * where the former has to be distinguished from any other arbitrary value.
+ * It can be thought of as an alternative to a type hint.
+ *
```

Comment:
I need to improve the doc. The meaning is basically this - in untyped data structure (e.g. JSON array, but also Array[Any]) there is no way to distinguish between `42` as just a number, attribute ID, expression ID, or anything else that this number could represent. For that any sort of type hint mechanism has to be used. This class serves exactly that purpose - a wrapper that marks the value as something special (in this case attr ID or expr ID).

Reply:
Changed.
I didn't mention Swagger there because it has nothing to do with that. It's not a Swagger issue we are dealing with but rather a limitation of our particular data model we have chosen and the way it is represented in semi-structured data formats (it doesn't even have to be JSON in particular).
Anyway I hope the doc is clearer now.